codes/9184_py3_sjskgks.py

```diff
@@ -0,0 +1,27 @@
+import sys
+input = sys.stdin.readline
+
+def memoization(a, b, c):
+    if log[a][b][c]!=-1:
+        pass

```

Comment:
pass는 아무것도 하지 않습니다.
https://docs.python.org/ko/3/tutorial/controlflow.html
바로 return하는게 더 좋아보입니다.
```suggestion
        return log[a][b][c]
```